src/component/ActionMappingForm.tsx

```
@@ -1,225 +1,377 @@
-    uid: UID
-} & ActionMap
-type ActionMapWithUIDList = ActionMapWithUID[];
 type ActionMapKey = "switchType" | "switchId" | "pluginId" | "actionId";
-
+type MappingList = Array<[string, string]>; // [uuid, presetName]
 
 export default function ActionMappingForm(props: {
-    portName: string
+    // actionMapPresets?: ActionMapPreset[];
+    // onSubmit: (e: ActionMapPreset) => void;
 }) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)